lib/route.js

```diff
@@ -304,15 +304,24 @@ function buildRouting (options) {
           // be available after all of the plugins and routes have been loaded.
           fourOhFour.setContext(this, context)
 
+          if (opts.validatorCompiler) {
+            context.config.validatorCompiler = opts.validatorCompiler
+          }
+
+          if (opts.serializerCompiler) {
+            context.config.serializerCompiler = opts.serializerCompiler
+          }
```

Comment:
I'm not sure all the changes in this file are needed because the reply.js's `setupValidator` call should do the job already

I will check

Reply:
In this case does not work with the 324/334 line addition?